apps/artemis-web/pages/hijack.tsx

```diff
@@ -0,0 +1,408 @@
+import {
+  FormControlLabel,
+  FormGroup,
+  Grid,
+  makeStyles,
+  Paper,
+  Switch,
+} from '@material-ui/core';
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import React, { useState } from 'react';
+import 'react-toastify/dist/ReactToastify.css';
+import NotFoundHOC from '../components/404-hoc/404-hoc';
+import BGPTableComponent from '../components/bgp-table/bgp-table';
+import { Editor, EditorState } from 'draft-js';
+import 'draft-js/dist/Draft.css';
+import { useGraphQl } from '../utils/hooks/use-graphql';
+import { formatDate, fromEntries } from '../utils/token';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+}));
+
+const ViewHijackPage = (props) => {
+  const [isLive, setIsLive] = useState(true);
+  const isDevelopment = process.env.NODE_ENV === 'development';
+  const isBrowser = typeof window !== 'undefined';
+
+  if (isDevelopment && isBrowser) {
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const { worker } = require('../utils/mock-sw/browser');
+    worker.start();
+  }
+
+  const statuses = {
+    Ongoing: 'danger',
+    Dormant: 'secondary',
+    Resolved: 'success',
+    Ignored: 'warning',
+    'Under Mitigation': 'primary',
+    Withdrawn: 'info',
+    Outdated: 'dark',
+  };
+
+  const findStatus = (row) => {
```

Comment:
There can be multiple statuses like withdrawn+outdated, etc
These are the possible states:
https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/other/db/data/tables.sql#L83